src/hooks/v2/NftRewards.ts

```diff
@@ -19,10 +21,373 @@ export default function useNftRewards(
       return
     }
 
-    const getRewardTiers = async () => {
+    const getRewardTiers = () => {
       return Promise.all(CIDs.map(cid => getRewardTierOfCid(cid)))
     }
 
     return getRewardTiers()
```

Comment:
**question** why not return the Promise.all directly instead?

Reply:
Yep can do